CHANGELOG.md

```diff
@@ -1,4 +1,27 @@
 ## 2.13.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.4.1045** 
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. Implemented support for ``enable_learned_cidrs_approval`` in **aviatrix_transit_gateway**, **aws_tgw_vpn_conn** and **aws_tgw_directconnect**
+2. Implemented a new parameter ``manage_transit_gateway_attachment`` to provide option of attaching/detaching transit gateway to/from AWS tgw outside of **aviatrix_aws_tgw**
```

Comment:
2. Implemented a new parameter ``manage_transit_gateway_attachment`` to provide the option of attaching/detaching the transit gateway to/from the AWS TGW outside of the **aviatrix_aws_tgw** resource